webroot/css/toolbar.css

```diff
@@ -512,10 +512,13 @@ pre,
   position: relative;
   top: 2px;
 }
-
 .btn-primary:hover {
   cursor:pointer;
 }
+.toggle-plugin-route-active {
+  background-color: #fff;
+  color: #2a6496;
```

Comment:
I'm not sure I love these colours in this context but I can fiddle with it afterwards.